charts/kubefed/values.yaml

```diff
@@ -6,19 +6,36 @@
 ##
 controllermanager:
   enabled: true
-  replicaCount: 2
-  repository: quay.io/kubernetes-multicluster
-  image: kubefed
-  tag: canary
-  imagePullPolicy: IfNotPresent
-  logLevel: 2
-  resources:
-    limits:
-      cpu: 500m
-      memory: 512Mi
-    requests:
-      cpu: 100m
-      memory: 64Mi
+  manager:
+    replicaCount: 2
+    repository: quay.io/kubernetes-multicluster
+    image: kubefed
+    tag: canary
+    imagePullPolicy: IfNotPresent
+    logLevel: 2
+    env: {}
+    resources:
+      limits:
+        cpu: 500m
+        memory: 512Mi
+      requests:
+        cpu: 100m
+        memory: 64Mi
+  webhook:
+    repository: quay.io/kubernetes-multicluster
+    image: kubefed
+    tag: canary
+    imagePullPolicy: IfNotPresent
+    logLevel: 8
+    env: {}
+    resources:
+      limits:
+        cpu: 100m
+        memory: 256Mi
+      requests:
+        cpu: 100m
+        memory: 64Mi
+
   clusterAvailableDelay:
```

Comment:
Aren't these properties also applicable only to `controller`?

Reply:
These are part of `controllermanager` main chart, perhaps I can move them up to easily identify the scope for manager` and `webhook`.

Reply:
I moved them up so it is more clear the purpose of the properties manager and webhook. To me, properties such as `clusterAvailableDelay` are defining a more generic behavior to the whole kubefed control-plane.